### PR TITLE
Set node/edit to use admin theme (Gin).

### LINKED ISSUE #86: Admin theme permissions and for content editing
As 'microsite admin' 
Editing a node on microsite 1
I want to see the admin theme.

Permissions and theme settings need updating.

### PR DESCRIPTION
**Pull request by finnlewis**

### CODE REVIEW

**Comment by finnlewis** [comment]
This look ok @ekes ?

**Comment by ekes** [comment]
So this has got the permission for media in it too?

And those permissions allow anyone to delete or edit the media from any other user/site?

File: config/install/user.role.authenticated.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    - fontawesome
+    - media
+    - system
+id: authenticated
+label: 'Authenticated user'
+weight: 1
+is_admin: false
+permissions:
+  - 'access content'
+  - 'access fontawesome additional settings'
```
Comment (ekes) [inline config/install/user.role.authenticated.yml]: Do they need this too?

File: config/install/user.role.authenticated.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  - 'access media overview'
+  - 'administer media'
+  - 'create document media'
+  - 'create image media'
+  - 'create media'
+  - 'create remote_video media'
+  - 'delete media'
```
Comment (ekes) [inline config/install/user.role.authenticated.yml]: Guessing no one should have access to delete media really, just remove it from your content.

File: config/install/user.role.authenticated.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  - 'delete own document media'
+  - 'delete own image media'
+  - 'delete own remote_video media'
+  - 'edit any document media'
```
Comment (ekes) [inline config/install/user.role.authenticated.yml]: I can see we might need to do something for editing media that is attached to content that is attached to a group that isn't _owned_ by the user, but again not sure this is the way or needs to be done now.

File: config/install/user.role.authenticated.yml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  - 'edit any image media'
+  - 'edit any remote_video media'
+  - 'edit own document media'
+  - 'edit own image media'
+  - 'edit own remote_video media'
+  - 'update any media'
+  - 'update media'
+  - 'use text format wysiwyg'
+  - 'view all media revisions'
```
Comment (ekes) [inline config/install/user.role.authenticated.yml:32]: We've also generally been hiding revisions to keep things simpler so my guess is we don't want revisions here either.

**Review by ekes — CHANGES_REQUESTED** [review]

**Comment by ekes** [comment]
I'm guessing there's another issue about media permissions somewhere that should also be tagged?

**Comment by finnlewis** [comment]
@ekes I removed those permissions. Looking better?

**Review by ekes — APPROVED** [review]
Looks better. Can we tag an issue to review the media permissions at some point though? I assume there is one, but if not spin out a follow-up.

**Comment by finnlewis** [comment]
Sorry - just tested again - it does need 'edit any image media'

**Comment by finnlewis** [comment]
> I'm guessing there's another issue about media permissions somewhere that should also be tagged?

https://github.com/localgovdrupal/localgov_microsites/issues/88

**Pull request merged**